src/Cache/Interceptors/AgentInterceptor.php

```diff
@@ -7,17 +7,23 @@
 
 class AgentInterceptor extends Interceptor
 {
-
-    private const AGENT = "php:0.5.0";
     private bool $isFirstRequest = true;
+    private string $agent;
+    private string $runtimeVersion;
+
+    public function __construct(string $clientType)
+    {
+        $this->agent = sprintf("php:%s:0.5.0", $clientType);
+        $this->runtimeVersion = PHP_VERSION;
+    }
 
     public function interceptUnaryUnary($method, $argument, $deserialize, $continuation, array $metadata = [], array $options = [])
     {
         if ($this->isFirstRequest) {
-            $metadata["agent"] = [self::AGENT];
+            $metadata["agent"] = [$this->agent];
```

Comment:
going off what I see here in nodejs

https://github.com/momentohq/client-sdk-javascript/blob/f3fa909eb2da64fe3ba17dc3fd3e5981bd7ce8cf/packages/client-sdk-nodejs/src/internal/pubsub-client.ts#L79-L81

it looks like these should be `Agent` and `Runtime-Version`

Reply:
Chris had this PR in Ruby where he mentioned headers must be lower case: https://github.com/momentohq/client-sdk-ruby/commit/606700265113c06c36ea6c603418c606101ddbb6. 
So, I left the headers the way there previously?

Reply:
hmmm interesting, either way we should change `runtime_version` -> `runtime-version`. Maybe we need to change the javascript sdk to also be lowercase?

Reply:
I can make the runtime_version change. 
@cprice404 Lowercase/uppercase headers?

Reply:
Looks like lowercase is actually required, so what you have is perfect.